server/src/room.ts

```diff
@@ -0,0 +1,38 @@
+import Player from './player';
+
+export default class Room {
+
+    roomID: string; 
+    #players: Map<string, Player>;
```

Comment:
Why don't you just use Objects?

Why do we have to use Map?

Reply:
so that I can index it by playerID. e.g., playerID[1234].  Maps were already being used before, except we only had 1 map for the entire game server instead of having it in a Room class

Reply:
You can do it in Objects as well

Reply:
the alternative is to use a List of Player objects, but then every time we need to access one, we have to search for a Player object which contains the corresponding playerID, which feels more complex to me. 

especially since we only ever access a Player object by player id

Reply:
I guess JS kinda has this built in? something like

`myArray.find(item => item.id === 2);`

that might be less confusing to read than a bunch of maps

Reply:
Huh what do you mean?

We can do the exact same thing as Maps with Objects, so there is no difference in complexity.

Reply:
wdym with Objects, we're already using Objects. Do you mean a list of objects?

Reply:
just join discorrd idiot